scroll-animations-1/Overview.bs

```diff
@@ -835,12 +835,20 @@ spec: cssom-view-1; type: dfn;
 	as step 1 of [=update animations and send events=].
 
 	During step 7.14.1 of the [HTML Processing Model](https://html.spec.whatwg.org/multipage/webappapis.html#processing-model-8),
-	any created [=scroll progress timelines=] or [=view progress timelines=] are collected into an <dfn export>initially stale</dfn> set.
-	After step 7.14.1 if there are any [=initially stale=] timelines, they update their current time,
-	the set of [=initially stale=] timelines is cleared and
+	any created [=scroll progress timelines=] or [=view progress timelines=] are collected into a <dfn export>stale timelines</dfn> set.
+	After step 7.14 if any timelines' [=named timeline ranges=] have changed,
+	these timelines are added to the [=stale timelines=] set.
+	If there are any [=stale timelines=], they now update their current time and associated ranges,
+	the set of [=stale timelines=] is cleared and
 	we run and we run an additional step to recalculate styles and update layout.
-	As we only gather stale timelines during the first style and layout calculation,
+
+	Note: We check for layout changes after dispatching ResizeObservers
+	intentionally to take into account programatically sized elements.
+
+	Note: As we only gather stale timelines during the first style and layout calculation,
```

Comment:
Also worth pointing out that the reason for only gathering stale timelines during the first style+layout cycle is to avoid an infinite style update loop. It is still possible for the style to be stale at the end of the second cycle, if the animation is changing the height or width of the source or subject of a scroll-driven animation.  This can lead to animation jitter.  It is safe however, for the animation to change the size via a transform, since using non-transformed values for determination of the animation's progress.

Reply:
Isn't this infinite loop concern covered by the first paragraph? And the paragraph on line 867 points out that they will be stale until the next frame.

Reply:
ack